json-schema/schema.json

```diff
@@ -234,6 +427,84 @@
                   }
                 }
               },
+              {
+                "title": "Sentinel1",
+                "if": {
+                  "$ref": "#/definitions/types/Sentinel1"
+                },
+                "then": {
+                  "required": [
+                    "pl:black_fill",
+                    "pl:pixel_resolution",
+                    "pl:quality_category",
+                    "eo:cloud_cover",
```

Comment:
@tschaub it might be strange to require an EO field for Sentinel1, but AFAIK that is how it is produced upstream. LMK if you think we should keep/remove this

Reply:
Looks like you're right. I was looking at a spec that implied that we did provide cloud_cover for Sentinel1, but I'm not able to find it present anywhere in the data api